### PR TITLE
fix: correct composite action for creating issues

### PR DESCRIPTION
**Pull request by yrobla**
#### Summary

The action needed to clone the repo on an independent directory to be able to consume it, and some minor tweaks

#### Release Note

NONE

#### Documentation

NONE

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]

**Review by haydentherapper — APPROVED** [review]

File: .github/actions/create-issue/action.yaml [hunk 1]
```diff
@@ -13,13 +13,15 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Checkout code
-      uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
-
+    - name: Checkout this repo
+      uses: actions/checkout@v3
```
Comment (cpanato) [inline .github/actions/create-issue/action.yaml:17]:
```suggestion
      uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

**Review by cpanato — CHANGES_REQUESTED** [review]
small nit

**Comment by haydentherapper** [comment]
FYI this was closed

**Comment by haydentherapper** [comment]
@yrobla Can this be reopened?

**Comment by yrobla** [comment]
sorry, i deleted because i had conflicts in the repo, and removed it, as i was doing a lot of local testing. I will create a new PR

**Pull request closed without merging**